### ISSUE TITLE
Reported Accuracy for cross camera adaptation on which dataset

### ISSUE THREAD

**Issue by vikash0837** [kevinhkhsu/DA_detection#6]
Hi,
Could you please clarify that reported accuracy in Cross camera model is based on the validation set of CItyScapes dataset. Also about the visualization, are you using the stage 2 model out to generate the feature for t-SNE visualization
Thanks

**Comment by kevinhkhsu** [comment]
Yes, the reported accuracy in the cross-camera model is based on CityScapes' validation data. 

For visualization (Fig. 3 in our paper), we are using the non-adapted model to generate feature maps in order to show the distributions of the three domains before adaptation and that our synthetic dataset is distributed in between the source and target domains.

**Comment by vikash0837** [comment]
Thank you

**Comment by life97** [comment]
> Yes, the reported accuracy in the cross-camera model is based on CityScapes' validation data.
> 
> For visualization (Fig. 3 in our paper), we are using the non-adapted model to generate feature maps in order to show the distributions of the three domains before adaptation and that our synthetic dataset is distributed in between the source and target domains.

Which pre-trained weights will be used to customize the data set in the first phase of the training network? If you can reply, thank you very much.
